Gems/EMotionFX/Code/Source/Integration/Components/ActorComponent.cpp

```diff
@@ -621,6 +625,12 @@ namespace EMotionFX
                     const bool updateTransforms = AZ::RHI::CheckBitsAny(m_configuration.m_renderFlags, s_requireUpdateTransforms);
                     m_actorInstance->SetIsVisible(isInCameraFrustum && updateTransforms);
                 }
+#if defined(CARBONATED)
+                else
+                {
+                    m_actorInstance->SetIsVisible(true);
+                }
+#endif
```

Comment:
@carbonated-niko 
This is the only essential change that fixes the bug, the other two are optional.
Actually our code makes the player invisible. An alternative or complimentary change can be to modify CharacterComponent::UpdateCharacterVisibility() to avoid calling actorInstance->SetIsVisible(isVisible); with false for player.

Reply:
The code piece is removed, I added force update flag getter instead.
Please note that with the latest state the bug is not fixed by this PR alone. We need a game code change: remove redundant visibility update or involve the new flag getter.